frontend/app/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -67,6 +68,7 @@ export function HomeScreen() {
           <BorrowTable compact={compact} />
         </div>
         <EarnTable compact={compact} />
+        <YieldSourceTable compact={compact} />
```

Comment:
I think Michael wanted to keep the yield source table on the right, even if it unbalances the page. We could add an empty `<div />` above `<YieldSourceTable />`.